tests/ttnn/unit_tests/operations/test_bernoulli.py

```diff
@@ -50,42 +58,28 @@ def run_bernoulli(shape, in_dtype, out_dtype, device, is_out_alloc=False, comput
 
         c = Counter(tt_output_list)
         one_probs.append(c[1] / len(tt_output_list))
+    logger.info(f"one_probs={one_probs}")
 
     expected_one_prob = 0.5
     assert np.allclose(expected_one_prob, np.mean(one_probs), rtol=0.05)
 
 
-# fmt: off
 @skip_for_grayskull("Requires wormhole_b0 to run")
-@pytest.mark.parametrize("shape",
+@pytest.mark.parametrize(
+    "shape",
     [
         [2003],
         [500, 500],
         [1, 512, 2, 256],
     ],
 )
-@pytest.mark.parametrize("in_dtype",
-    [
-        "bfloat16",
-        "float32"
-    ]
-)
-@pytest.mark.parametrize("out_dtype",
-    [
-        "bfloat16",
-        "float32"
-    ]
-)
-@pytest.mark.parametrize("is_out_alloc",
-    [
-        True,
-        False
-    ]
-)
-# fmt: on
-def test_bernoulli(shape, in_dtype, out_dtype, device, is_out_alloc):
-    torch.manual_seed(0)
-    run_bernoulli(shape, in_dtype, out_dtype, device, is_out_alloc)
+@pytest.mark.parametrize("seed", [6296, 3501, 1712])
```

Comment:
Is it okay not to run tests when the seed is 0?

Reply:
It's fine, but I would rather allow seed = 0 than skip the test for seed = 0. This is because when the seed is set to 0 (causing the program factory to generate a random seed), running the tests 100 times - 100 random seeds still results in most of them passing.

Reply:
I asked because I noticed that the run_bernoulli function has code that runs when the seed is set to 0.
Would it make sense to either pass seed 0 as an input or remove that logic if it’s not needed?
```python
    iter_num = 1
    # When seed param is 0, a random seed is generated in the program factory, we run the test multiple times
    if seed == 0:
        iter_num = 10

```

Reply:
I planned to support seed = 0 in the test when the issue with tensix instruction is fixed. This may cause confusion, so I've just removed it as you suggested.

Reply:
Got it. If that issue is resolved, let’s go ahead and add it.